### PR TITLE
Currency formatting issue fixed and Add Transfer Icon Fix(Ticket 331 and 507)

### PR DESCRIPTION
**Pull request by aparthibanpaypal**

### CODE REVIEW

**Review by shanavas123 — APPROVED** [review]

**Review by rbanikPP — APPROVED** [review]

File: Transfer/Sources/TransferDestinationCell.swift [hunk 1]
```diff
@@ -89,7 +89,7 @@ extension TransferDestinationCell {
         if !UIFont.isLargeSizeCategory {
             let icon = UIImage.fontIcon(hyperwalletIcon.rawValue,
                                         Theme.Icon.frame,
-                                        CGFloat(Theme.Icon.size),
+                                        CGFloat(20),
```
Comment (rbanikPP) [inline Transfer/Sources/TransferDestinationCell.swift:92]: dont use hardcoded value. Create something like Theme.addIcon.size

File: Transfer/Sources/CreateTransferController.swift [hunk 2]
```diff
@@ -234,11 +234,9 @@ extension CreateTransferController {
         if let transferMethod = presenter.selectedTransferDestination {
             tableViewCell.configure(transferMethod: transferMethod)
         } else {
-            if selectTransferMethodCoordinator == nil {
-                tableViewCell.accessoryType = .none
-            }
+            tableViewCell.accessoryType = .none
```
Comment (rbanikPP) [inline Transfer/Sources/CreateTransferController.swift:237]: Why is this changed?

Comment (aparthibanpaypal) [reply Transfer/Sources/CreateTransferController.swift:237]: To hide the arrow mark next to Add Transfer Method title.

**Review by jsaini-hw — APPROVED** [review]

**Review by ddhar-hw — APPROVED** [review]

File: Transfer/Sources/TransferDestinationCell.swift (hunk 1 above)
Comment (aparthibanpaypal) [reply Transfer/Sources/TransferDestinationCell.swift]: Added constant
